src/scenes/TradeHub/index.js

```diff
@@ -48,14 +49,25 @@ export default class TradeHub extends Component {
 
   onDismiss = () => {
     const alert = Object.assign({}, this.state.alert);
-    alert.isOpen = false;
+    alert.open = false;
     this.setState({ alert });
-  }
+  };
+
+  setAlert = (open, message, color) => {
```

Comment:
This micro API is vague and unnecessary. You're better of without it, really

Reply:
When removing this method, there will be much duplicated code to implement the same functionality.

Reply:
No there won't be since you're essentially just setting the state.
You could replace a call to `setAlert` like this:
```javascript
setAlert(true, 'Foobar', 'warning');
```
with this:
```javascript
this.setState({
  alert: {
    open: true,
    message: 'Foobar',
    color: 'warning',
  },
});
```
This is longer but *much, much much* more clear